src/bin/vip-import-sql.js

```diff
@@ -573,7 +573,7 @@ Processing the SQL import for your environment...
 		} catch ( uploadError ) {
 			await track( 'import_sql_command_error', {
 				error_type: 'upload_failed',
-				upload_error: uploadError,
+				upload_error: uploadError.message,
```

Comment:
We used to throw `string` from the code (OMG). I refactored the code to throw `Error` instead.